src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -66,7 +65,7 @@ private static String getPackageFromFilePath(String filePath) {
     private static List<String> readFile(Path filePath) throws CheckstyleException {
         try (BufferedReader br = Files.newBufferedReader(
                 filePath, StandardCharsets.UTF_8)) {
-            return new ArrayList<>(br.lines().collect(Collectors.toList()));
+            return br.lines().collect(Collectors.toList());
```

Comment:
`return Files.readAllLines(filePath)` would be even simpler

Reply:
done